eth-contracts/contracts/Governance.sol

```diff
@@ -332,7 +335,9 @@ contract Governance is RegistryContract {
             outcome = Outcome.No;
         }
 
-        // Record outcome
+        /// Re-entrancy should not be possible here since the function switches the status of the
+        /// proposal to 'Evaluating' so it should fail the status is 'InProgress' check. This
+        /// records the final outcome in the proposals mapping
```

Comment:
This is a good comment - but imo should be shifted above to the "evaluating" value change